Source/Core/Core/ConfigManager.cpp

```diff
@@ -325,7 +325,7 @@ void SConfig::SaveNetworkSettings(IniFile& ini)
 
   network->Set("SSLDumpRead", m_SSLDumpRead);
   network->Set("SSLDumpWrite", m_SSLDumpWrite);
-  network->Set("SSLVerifyCert", m_SSLVerifyCert);
+  network->Set("SSLVerifyCertificates", m_SSLVerifyCert);
```

Comment:
TBH, I think I was probably one of the few using it and if no body complained about that lack of SSL verifications since the last 2 years. It seems perfectly safe just to change the default value.

Knowing that most lambda users don't care about that feature and I don't think that the cleanest way to reset the default value. In other words, the "Network" entry shouldn't be present in their configuration file. All my portable builds of Dolphin after that addition don't have it so unless that behavior was added recently it shouldn't be the case.